File: 2012_1_5__client_side_mvx_frameworks/SampleApp/SampleApp/Code/Card.cs
using System;

namespace SampleApp.Code
{
    public class Card
    {
        public int id { get; set; }
        public string recto { get; set; }
        public string verso { get; set; }
        public int correctCount { get; set; }
        public int incorrectCount { get; set; }
        public bool isArchived { get; set; }
        public int userId { get; set; }
    }

}

File: 2012_1_5__client_side_mvx_frameworks/SampleApp/SampleApp/Content/models/card.js
(function (exports) {

    var Card = Spine.Model.setup('Card', ['recto', 'verso', 'isArchived']);
    Card.extend(Spine.Model.Ajax);
    Card.extend({
        url: '/api/card'
    });
    Card.include({
        validate: function () {
            if (!this.recto) return "Text for the front of the card is required.";
            if (!this.verso) return "Text for the back of the card is required.";
        }
    });
    Card.bind('refresh', function () {
        console.log('refreshing', Card.all());
    });

    exports.Card = Card;

})(this);